OpenRA.Mods.Common/Traits/Player/ClassicProductionQueue.cs

```diff
@@ -120,19 +120,20 @@ protected override bool BuildUnit(ActorInfo unit)
 
 		public override int GetBuildTime(string unitString)
 		{
-			return GetBuildTime(self.World.Map.Rules.Actors[unitString]);
+			var actorInfo = self.World.Map.Rules.Actors[unitString];
```

Comment:
This overload of `GetBuildTime` is only called from one place, so you might as well inline this lookup there and remove the method.